analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -124,42 +142,59 @@ def create_learners(self, learners):
         self._es.indices.refresh(index=settings.ELASTICSEARCH_LEARNERS_INDEX)
 
 
+@ddt.ddt
 class LearnerTests(VerifyCourseIdMixin, LearnerAPITestMixin, TestCaseWithAuthentication):
     """Tests for the single learner endpoint."""
     path_template = '/api/v0/learners/{}/?course_id={}'
 
-    def setUp(self):
-        super(LearnerTests, self).setUp()
+    @ddt.data(
+        ('ed_xavier', 'Edward Xavier', 'edX/DemoX/Demo_Course', 'honor', ['has_potential'], 'Team edX',
+         43, 3, 6, 0, 8.4, 2, '2015-04-24', '2015-08-05'),
+        ('ed_xavier', 'Edward Xavier', 'edX/DemoX/Demo_Course', 'verified', None, None,
```

Comment:
Instead of passing a bunch of `None`s here you can set default parameter values in the function signature.